FastBitSet.js

```diff
@@ -71,7 +79,7 @@ FastBitSet.prototype.flip = function (index) {
 
 // Remove all values, reset memory usage
 FastBitSet.prototype.clear = function () {
-  this.words = [];
+  this.words.length = 0;
```

Comment:
It seems here that you are changing the semantics. The 'clear' function is documented as something which "resets the memory usage".

Reply:
The description of `Array.length` seems to imply that is what it does.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/length#Description

It looks like setting the length truncates or allocates a chunk of memory.

Reply:
Sounds reasonable.